contents/handbook/company/security.md

```diff
@@ -6,16 +6,29 @@ showTitle: true
 
 It is critical that everyone in the PostHog team follows these guidelines. We take people not following these rules very seriously - it can put the entire company and all of our users at risk if you do not.
 
-## Password managers
-
-You **must** make use of a password manager; it simply isn't possible to use appropriate passwords securely without one.
-
-PostHog uses [1password](https://1password.com/) for storing all passwords.
-
-## Password strength
-
-Please use strong passwords for everything. Use the 1password password generator that comes with the app in all cases. Do not repeat passwords across different sites. 
-
-## Two-factor authentication
-
-You should enable two-factor authentication for any account where the option is available, especially those which are core to your work. 
+## Security policies
+
+We are in the process of obtaining our SOC 2 certification, which has required us to put together a number of (short!) policies to ensure compliance. You will have been invited to Drata to review these and to complete security training as part of your onboarding.  
+
+All of our policies can be found in our Drata portal, so this section of the Handbook just serves to make these policies publicly available in case you need to refer back quickly, or if a customer asks. These are only linked as PDFs so we only need to keep the policies up to date in Drata. 
+
+- [Acceptable Use Policy](https://dt-prod-app-us-west-2.s3.us-west-2.amazonaws.com/805315dd-8452-461b-850d-fb1957ecb803/policies/42d32ae6-c90f-4f82-87c7-03ca8910eb84/AcceptableUsePolicy.pdf?AWSAccessKeyId=ASIAT5KN2HOP5ZGK22X6&Expires=1621337921&Signature=ehBha%2BTIqi9oUc8uClbdJoz7LW8%3D&x-amz-security-token=IQoJb3JpZ2luX2VjEL7%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLXdlc3QtMiJHMEUCIQDkZCPRCfprrJaPjs6ck8VhLFS%2F7HJp3CLeoHou3Axg%2FAIgHb1izGS2W0J6Hg81gi6HeiCeWJyGvPsFn2%2BmRrpALvEq9QMIVxAAGgwyNjkxMzU1MjY4MTUiDNXCO8Gtzt0T9CrMXSrSA2hVZva%2BGMIZC%2BSjRD%2FHUYrS%2FVN22Bs%2FbYXr4whHpJZhc5wHS%2BiesYHGcrieMNh6NCxbzRM%2FcmHNFrO5ve2sGbTYpJKSgUI%2F%2BJrRtLL9gzOZ%2Fgm3kphozieveA%2FzhQi%2FPwTnj9vnwEXsIdWTsDWbl4PPde%2BZlqFBcVyBOnZZbXs8ExboOxguAjmusehwADxC4VG%2FnyncG8uGezZstbiggDbbiKcbLy9fjr4FSKXDp10PfjP1hoBlGl2giSqRtClUXrWsetPCEt943YL%2FJG96Uv5FW3WwArpTHrXQSTq8LikE1YPIco9lAgZmqjTzvIvXjNo6CjA0GuhhYOMZTKRTKaAkyNqMPxjUt4QO%2FPXGFIrBHa%2BfRcu1LMqAx0%2FQymtDB6tHwPI9b6HzA230dkJTRIv4ZzM7%2FkJ%2BJ8YDNv7tiZ6wd9xYqMrWVAxeAmL%2FHtAAHne97N25ZoCDQ78D2bE1azRduLggmQv7VTvgM2o6yHiw42iNTrwJIaEHpMptID81%2FCm13Gfnxov8hnmwIF3FvFnmVIjguhlqW%2F8C3LyJodJVAxoTq3XPc1gAfHFAUlCXJpuXR8BPxiHhJhz9fePvOd4LNXlMFr7%2FG%2FBnI9McD4fgbM4wjbWNhQY6pQFStyBbMinonf%2FTQjppTLj6Nmf4wA0S3EA0EipUN4WMqvBfCxD1kSpFb0eyHdPyOsxOqHKn9A9tD3t%2BA5EdcTaQbDfFoM49H83VDVYVzcPHkSbiMPTDZSm7ua98grUqRS4KbuKYwZSr7DBeXY%2Fdbqz0gsxVfS7QmTTIJiT2ntGAuVP3UXdJgvrLV4meK0eklqH9nLQtOBJFzCiSxWL6go6bDOO4Oog%3D)
```

Comment:
These links don't load for me if our aim is to make this public

![Uploading Screenshot 2021-05-18 at 13.52.01.png…]()

Reply:
Blah, yes I tested in incognito and they worked for some reason. I'll fix it another way.

Reply:
@jamesefhawkins should be fixed now